.github/workflows/bench.yml

```diff
@@ -45,6 +47,39 @@ jobs:
         cabal-version: ${{ matrix.cabal }}
         enable-stack: false
 
+    - if: runner.os == 'Windows'
```

Comment:
Yes. Tried to keep the same code between workflows.

It is an interesting question :thinking:. From the side of code sharing, it is seamless, generalized & not "overfitted" to local code, but from the workflow point of view it can be said to have superfluous features, it sounds like something covered in some book.

Reply:
hmm if we got to reuse steps between workflows it would be needed but i would prefer to not have unused steps even losing the overall consistency.